### DISCUSSION TITLE
Central Boiler ON/OFF plus several smartvalve on heaters  - Which is the best approach/configuration

### DISCUSSION THREAD

**Discussion by andreapede** [jmcollin78/versatile_thermostat#557] [Q&A]
Hi, 
in my house I have a central boiler that I can switch on/off, a temperature sensor in each room and over the heater on this room also a smart valve (sonoff) that i can control. Which is the best configuration that I can apply? Rooms have a different temperature coefficient being exposed in different area.

**Comment by jmcollin78** [comment]
Hello @andreapede ,

If the smart valve SONOFF can be directly controlled with a number entity which controls the open percentage, the best solution is to create an `over_valve` VTherm with control of a central boiler available in the central confiugration.

See README for all this points if it is not clear.

If you cannot control the valve direclty, you will have to create an `over_climate` VTherm which is not the best solution ton control a central boiler with VTherm. It depends if your SONOFF climate, have a reliable hvac_action which is "heating" if the TRV is open.
